types/group.pp

```diff
@@ -0,0 +1,2 @@
+# Valid $user parameter to Cron::Job.
```

Comment:
This file was added but not used.

Reply:
Sorry about that.  I'm about four years rusty and am struggling to remember all thing things that need to be done.  Fixing those things now.